File: library/src/main/java/com/senierr/sehttp/https/UnSafeHostnameVerifier.java
package com.senierr.sehttp.https;

import javax.net.ssl.HostnameVerifier;
import javax.net.ssl.SSLSession;

/**
 * 非安全主机名验证，默认所有可接受
 *
 * @author zhouchunjie
 * @date 2017/8/25
 */
public final class UnSafeHostnameVerifier implements HostnameVerifier {

    @Override
    public boolean verify(String hostname, SSLSession session) {
        return true;
    }
}

File: rxhttp/src/main/java/com/senierr/http/https/UnSafeHostnameVerifier.kt
package com.senierr.http.https

import javax.net.ssl.HostnameVerifier
import javax.net.ssl.SSLSession

/**
 * 非安全主机名验证，默认所有可接受
 *
 * @author zhouchunjie
 * @date 2017/8/25
 */
class UnSafeHostnameVerifier : HostnameVerifier {

    override fun verify(hostname: String, session: SSLSession): Boolean {
        return true
    }
}
